content/docs/setup/kubernetes/download/index.md

```diff
@@ -51,3 +51,51 @@ To use the Istio release Helm chart repository, add the Istio release repository
 {{< text bash >}}
 $ helm repo add istio.io https://storage.googleapis.com/istio-release/releases/{{< istio_full_version >}}/charts/
 {{< /text >}}
+
+## `istioctl` tab completion
+
+If you are using Bash, `istioctl` includes a helpful script that enables tab completion for the available `istioctl` commands.
+
+### Prerequisite for macOS
+
+If you are using macOS with the Bash shell, make sure that the bash-completion package is installed. If you are using the [Homebrew](https://brew.sh) package manager for macOS, you can check to see if the bash-completion package is installed with the following command:
+
+{{< text bash >}}
+$ brew info bash-completion
+{{< /text >}}
+
+If installed, the output from this command should display output similar to the following:
```

Comment:
... command should be similar to:

Reply:
Just combine these two text blocks into one, no value in repeating the command-line twice.